src/internal/connector/exchange/attachment.go

```diff
@@ -53,6 +53,12 @@ func uploadAttachment(
 		return nil
 	}
 
+	// item Attachments to be skipped until the completion of Issue #2353
+	if attachmentType == models.ITEM_ATTACHMENTTYPE {
+		logger.Ctx(ctx).Debugf("skip uploading of item attachment. Not supported: ", *attachment.GetName())
```

Comment:
@dadams39 - let's make this an Info log. @ryanfkeepers - what "clues" would be useful to add here? You'd want the user to be able to identify exactly which email they hit this issue with

Reply:
`logger.Ctx(ctx)` will import all clues currently in the context automatically.  That means if the caller already set the item ID into the context, it gets inherited here.  For ad-hoc values, I'd suggest the following:
```suggestion
		logger.Ctx(ctx).Debugw("item attachment uploads are not supported ",
		    "attachment_name", *attachment.GetName(),
		    "attachment_type", attachmentType,
		    "attachment_id", *attachment.GetId())
```

Reply:
Resolved with [962aaf8](https://github.com/alcionai/corso/pull/2359/commits/962aaf895ac48e8130d6ad2cac4b3f787153c56a)